hipcheck/src/main.rs

```diff
@@ -503,12 +506,24 @@ fn cmd_plugin(args: PluginArgs) {
 		working_dir: working_dir.clone(),
 		entrypoint: entrypoint2.display().to_string(),
 	};
+	let exec_config = if let Some(p) = config.exec() {
+		ExecConfig::from_file(p)
+		.context("Failed to load the exec config. Please make sure the exec config file is in the provided location and is formatted correctly.")
+	} else {
+		Err(hc_error!(
+			"No exec file found. Please provide an exec config file before running Hipcheck."
+		))
+	};
+
+	let plugin_data = exec_config.unwrap().plugin_data;
```

Comment:
Instead of unwrapping, change the function to return an `ExitCode` and use `Shell::print_error()` like `cmd_setup()` and `cmd_check()`